tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -119,17 +130,21 @@ spec:
       aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
       cat $(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml
       ENABLE_EXEC_SERVICE=false /clusterloader --kubeconfig=/root/.kube/config --testconfig=$(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml --testoverrides=$(workspaces.source.path)/overrides.yaml --nodes=$(params.nodes) --provider=eks --report-dir=$(workspaces.results.path) --alsologtostderr --v=2 $CL2_PROMETHEUS_FLAGS
-      if [ $? -eq 0 ]; then
+      exit_code=$?
+      if [ $exit_code -eq 0 ]; then
       echo "1" | tee $(results.datapoint.path)
       else
       echo "0" | tee $(results.datapoint.path)
       fi
+      exit $exit_code
     timeout: 30000s
   - name: upload-results
     image: amazon/aws-cli
     workingDir: $(workspaces.results.path)
     script: |
+      S3_RESULT_PATH=$(cat $(workspaces.source.path)/s3_result_path.txt)
```

Comment:
So the idea here is to add s3 result path as prometheus label for easy visualization of metrics. Since the result path is computed dynamically in the Tekton task using timestamp https://code.amazon.com/packages/EKSScalabilityTektonTestRuns/blobs/685d1c87e58fd4525ae50c177ee12301ff58ebfa/--/loadtests/triggers/eks/awscli-5k-load.yaml#L88. We are computing the s3 result path here and the same is passed to the next step so that the same path is used there also. Recomputing the result path on next step can result in a different path so it passed via file.

Reply:
I know we use timestamp in `params.results-bucket`. But I expect it to be fixed once initialized.
Are you saying `params.results-bucket` is not fixed. I'd surprised if that's the case.

Reply:
have started tekton result object for passing the values to next step instead of file.

Reply:
So you are saying `$(date +%s)` will be evaluated in every step that uses it. This is interesting to learn.

>have started tekton result object for passing the values to next step instead of file.

Yeah, that's something I'd suggest. It's definitely light-weight than communicating through s3.
We are using it in the CloudWatch emitting metrics tasks as well.